use_cases/eluc/app/utils.py

```diff
@@ -106,27 +72,17 @@ def create_check_options(values: list) -> list:
              "value": val})
     return options
 
-
-def compute_percent_change(context: pd.Series, presc: pd.Series) -> float:
+def context_presc_to_df(context: pd.Series, presc: pd.Series) -> pd.DataFrame:
```

Comment:
This is a new function to get the demo to work with the new prescriptors. Since the prescriptor wants a context_actions dataframe, we have to convert the context which we get from our dataset and the prescriptions which we get from our sliders into a context_actions_df by computing the diff and processing the zero change columns.